io_scheduling/scheduler.py

```diff
@@ -0,0 +1,11 @@
+import random as r
+
+class Scheduler:
+	def schedule(queue):
+		return 0
+
+
+
+class Random_scheduler(Scheduler):
+	def schedule(queue):
+		return r.randint(0,len(queue))
```

Comment:
it is expected to change the queue (remove the item index by the r.randint(0,len(queue)) (if the queue is not empty)